gptcache/adapter/diffusers.py

```diff
@@ -0,0 +1,48 @@
+from collections import OrderedDict
+import base64
+from io import BytesIO
+
+import diffusers
+from gptcache.adapter.adapter import adapt
+from gptcache.manager.scalar_data.base import Answer, AnswerType
+from gptcache import CacheError
```

Comment:
use `from gptcache.util.error import CacheError`, because `from gptcache import CacheError` will be deleted